helm-charts/mock-bot/values.yaml

```diff
@@ -80,4 +80,11 @@ appSecrets:
   # Also reference the secret key from the secrets object e.g.
   # APP__TESTSECRET:
   #   secretName: app-appsettings-secrets
-  #   secretKey: testKey
\ No newline at end of file
+  #   secretKey: testKey
+  CONNECTIONSTRINGS__APIKEY:
```

Comment:
Great stuff here, assuming the values are in the key vault we might need to set them in the reusable-k8s-deployment.yml pipeline 'deploy to Kubernetes' step?

Reply:
Ah yes.